CONTRIBUTING.md

```diff
@@ -66,28 +66,40 @@ Then every time you want to test a new version, you can just:
 ```shell
 make oci
 # Delete the mcoa pod which will make the Deployment pull the new image
-oc -n open-cluster-management delete pod -l app=multicluster-observability-addon-manager
+oc -n open-cluster-management-observability delete pod -l app=multicluster-observability-addon-manager
 ```
 
-### Disabeling specific signals 
+### Enable specific Observability Capabilities
 
-The addon supports disabling signals using the resource `AddOnDeploymentConfig`. For instance, to disable the logging signal create the following resource on the hub cluster:
+The addon supports enabling observability capabilities using the resource `AddOnDeploymentConfig`. For instance, to enable platform and user workloads logging/tracing/instrumentation create the following resource on the hub cluster:
 
 ```yaml
 apiVersion: addon.open-cluster-management.io/v1alpha1
 kind: AddOnDeploymentConfig
 metadata:
   name: multicluster-observability-addon
-  namespace: open-cluster-management
+  namespace: open-cluster-management-observability
 spec:
   customizedVariables:
-    - name: loggingDisabled
-      value: "true"
+  # Platform Observability
+  - name: platformLogsCollection
+    value: clusterlogforwarders.v1.logging.openshift.io
+  # User Workloads Observability
+  - name: userWorkloadLogsCollection
+    value: clusterlogforwarders.v1.logging.openshift.io
+  - name: userWorkloadTracesCollection
+    value: opentelemetrycollectors.v1alpha1.opentelemetry.io
+  - name: userWorkloadInstrumentation
+    value: instrumentations.v1alpha1.opentelemetry.io
 ``` 
 
-Supported keys are `metricsDisabled`, `loggingDisabled` and `tracingDisabled`
+Supported keys are:
```

Comment:
could we document the semantical meaning of the value? Especially that it can be a list?